.github/workflows/uberjar.yml

```diff
@@ -252,20 +251,23 @@ jobs:
         run: while ! curl -s 'http://localhost:3001/api/health' | grep '{"status":"ok"}'; do sleep 1; done
         timeout-minutes: 3
       - name: Login to Docker Hub
+        if: ${{ github.ref_name == 'master' }}
         uses: docker/login-action@v2
         with:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
       # Push experimental ubuntu image only for versions based on a master
       - name: Install regctl
+        if: ${{ github.ref_name == 'master' }}
         uses: regclient/actions/regctl-installer@main
         with:
           release: 'v0.4.7'
       - name: Switch regctl to point to localhost:5000 via http
+        if: ${{ github.ref_name == 'master' }}
         run: regctl registry set --tls disabled localhost:5000
       - name: Retag and push ubuntu-based images if master (ee)
-        if: ${{ matrix.edition == 'ee' }}
+        if: matrix.edition == 'ee' && github.ref_name == 'master'
```

Comment:
For consistency should we either always use `${{ }}` or not at all in the `if: ` clause?

[From here](https://docs.github.com/en/actions/learn-github-actions/expressions), it said we could optionally omit `${{ }}` in if clause.

Reply:
@WiNloSt take another look, please
https://github.com/metabase/metabase/pull/35737/commits/095e7a2678c23d0731abcff0414af0c8a267db6e

I prefer to use the syntax without `${{ ]}` but that would require changing the whole workflow so I brought them back for these two lines only.

Reply:
Awesome. Thanks for addressing this 🙏